staff/tomas-sancheznavarro/cv/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>Juan de los Palotes | Web Developer</h1>
+        <hr>
+    </header>
+
+    <address>
+        Carrer de la Unió 163<br>Premià de Mar<br>Barcelona<br>
+        <a href="mailto:palotes.juan@jeemail.com">palotes.juan@jeemail.com</a>.<br>
+        <hr>
+    </address>
+
+    <section id="goals">
+        <h2><u>Goals</u></h2>
+        <p>
+            Want fun, stimulating job that pays me in tacos and a mountain of money every month. Oh, and fifty days of
+            paid vacation. And a gym membership. Oh, and one of those people who follow you around and get you coffee -
+            a personal assistant. Yes, one of those.
+        </p>
+    </section>
+
+    <section id="skills">
+        <h2><u>Skills</u></h2>
+        <h3>Programming & Markup Languages</h3>
+        <ul>
+            <li>HTML5</li>
+            <li>CSS3</li>
+            <li>Javascript</li>
+        </ul>
+        <h3>Databases</h3>
+        <ul>
+            <li>MySQL</li>
+            <li>Mongo DB</li>
+            <li>Mongoose</li>
+        </ul>
+        <h3>Frameworks</h3>
+        <ul>
+            <li>React.js</li>
+            <li>Vue.js</li>
+            <li>Node.js</li>
+        </ul>
+    </section>
+
+    <section id="education">
```

Comment:
the id is not necessary, you must remove it

Reply:
done!